aten/src/TH/generic/THTensorEvenMoreMath.cpp

```diff
@@ -443,6 +443,125 @@ void THTensor_(indexFill)(THTensor *tensor, int dim, THLongTensor *index, scalar
   THLongTensor_free(index);
 }
 
+void THTensor_(gather)(THTensor *tensor, THTensor *src, int dim, THLongTensor *index)
```

Comment:
Is this just moving the code around in this file?

Reply:
yeah